.github/workflows/Linux.yml

```diff
@@ -16,31 +16,27 @@ jobs:
         include:
           - static_analysis: 0
             docs: 0
-            cmake:
+            cmake: 1
```

Comment:
Can this be 3.5.0 instead of 1. Perhaps we inject this value into the download path? We always use cmake.